eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -193,7 +194,7 @@ function Set-ChangeLogContent {
 
   try
   {
-    $ChangeLogEntries = $ChangeLogEntries.Values | Sort-Object -Descending -Property ReleaseStatus, ReleaseVersion
+    $ChangeLogEntries = $ChangeLogEntries.Values | Sort-Object -Descending -Property ReleaseStatus, ReleaseVersionSemVer
```

Comment:
What about the New-ChangeLogEntry code? We can also do the conversation inline as part of this sort via an expression. See https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/sort-object?view=powershell-7.1#example-5--use-a-hash-table-to-sort-properties-in-ascending-and-descending-order

Reply:
I switched to using the sorting expressiong.